index.ts

```diff
@@ -16,4 +25,11 @@ export default {
         validateBody,
         validateQuery,
     },
+    sleep,
+    convertTimeStringToSeconds,
+    randomElement,
+    roundToNearest,
+    deleteFiles,
+    decapitalizeFirstLetter,
+    randomRange,
```

Comment:
Comment:

I'd prefer if these were nested into their respective 'subModule'/folder name, similar to `validateBody` being put into `expressMiddlewares`. Declutters the namespace IMO.

```
const { AxiosDecorator, Lru } = require('@kgs-research/utils');
const { sleep, convertTimeStringToSeconds } = require('@kgs-research/utils').generic;
```

vs

```
const { AxiosDecorator, Lru, sleep, convertTimeStringToSeconds } = require('@kgs-research/utils');
```